Doppler.Currency/Program.cs

```diff
@@ -15,6 +16,11 @@ public static void Main(string[] args)
             Host.CreateDefaultBuilder(args)
                 .UseSerilog((hostContext, loggerConfiguration) => 
                     loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))
+                .ConfigureAppConfiguration(configHost =>
+                {
+                    configHost.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+                    configHost.AddKeyPerFile("/run/secrets", true);
+                })
```

Comment:
This is out of the scope of this PR. It is fine add it, but would better do it in another PR.